README.md

```diff
@@ -24,14 +24,18 @@ Unified API for music players.
 
 #### Linux
 
-- [ ] [MPRIS](https://specifications.freedesktop.org/mpris-spec/latest/)
+- [x] [MPRIS](https://specifications.freedesktop.org/mpris-spec/latest/) (test needed) (Thanks to [@suransea](https://github.com/suransea))
+
+##### dependencies
+
+- [playerctl](https://github.com/altdesktop/playerctl) (could be installed by package manager)
 
 #### Universal
 
 - SystemMedia: System-wide Now Playing
   - [x] macOS
   - [x] iOS (jailbroken device only) (test needed)
-  - [ ] Linux (is this possible?)
+  - [x] Linux (fake with MPRIS) (test needed)
```

Comment:
Put this under Helper section because it's not a Universal solution.
```suggestion
- [x] MPRIS Now Playing: Just like Now Playing, but automatically find available MPRIS players.
```